with-inbox-sdk/popup.tsx

```diff
@@ -0,0 +1,36 @@
+import { useEffect } from "react"
+import * as InboxSDK from '@inboxsdk/core';
+
+function IndexPopup() {
+  useEffect(() => {
+    InboxSDK.load(2, "YOUR_APP_ID", {}).then((sdk) => {
+      sdk.Compose.registerComposeViewHandler((composeView) => {
+        composeView.addButton({
+          title: "My Nifty Button!",
+          iconUrl:
+            "https://lh5.googleusercontent.com/itq66nh65lfCick8cJ-OPuqZ8OUDTIxjCc25dkc4WUT1JG8XG3z6-eboCu63_uDXSqMnLRdlvQ=s128-h128-e365",

```

Comment:
Use this image instead: https://www.plasmo.com/logo.png

Reply:
I will do that